src/deepEqual.js

```diff
@@ -24,7 +24,28 @@
  */
 
 function deepEqual(a, b) {
-  // write code here
+  const _isObj = arg => typeof arg === 'object' && arg !== null;
+  const _isKeys = arg => Object.keys(arg);
```

Comment:
this function returns keys. So it's name should be something like _getKeys.

To simplify the code, you could also do 
`const _getKeysLength = arg => Object.keys(arg).length; `

Reply:
Okay, thank you master ))